includes/class-wc-skroutz-analytics-tracking.php

```diff
@@ -248,4 +286,59 @@ private function calculate_tax_from_total($total, $tax_rate) {
 		return round( $total - $total / ( 1 + $tax_rate/100 ), 2 );
 	}
 
+	/**
+	 * Returns the paid by field to analytics
+	 *
+	 * @return string  The paid by field
+	 *
+	 * @since    1.7.0
+	 */
+	private function get_order_paid_by($payment_gateway) {
```

Comment:
[minor] Add whitespaces before and after the function arguments, to function definition and call.
[minor] Add @param phpdoc 

These apply to all the newly added functions